pkg/virt-launcher/virtwrap/network/bindings_test.go

```diff
@@ -0,0 +1,164 @@
+package network
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/mock/gomock"
+	"github.com/vishvananda/netlink"
+
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/pkg/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/cache/fake"
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+var _ = Describe("BindingMechanism", func() {
+	testRunPlug := func(driver BindMechanism, podInterfaceName string) {
```

Comment:
I think I would separate the tests for `discoverPodNetworkInterface` and `preparePodNetworkInterface`, not sure...
Anyway, I would change the term `Plug` since `Plug` is not in the tested file any-more. Maybe - `testDiscoverAndPrepare`

Reply:
Renamed to `testDiscoverAndPrepare`